### PR TITLE
changed "..." buttons to "Setup"

### PR DESCRIPTION
**Pull request by SteffenHeu**

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
Wouldn't it be better to use something like "Browse" or "Select" instead of "Setup"? We are not really setting up filenames.

File: src/main/java/io/github/mzmine/parameters/parametertypes/filenames/FileNameComponent.java [hunk 1]
```diff
@@ -53,7 +52,7 @@ public FileNameComponent(int textfieldcolumns, List<File> lastFiles, FileSelecti
     // on click - set file name to textField
     btnLastFiles = new LastFilesButton("last", file -> txtFilename.setText(file.getPath()));
 
-    Button btnFileBrowser = new Button("...");
+    Button btnFileBrowser = new Button("Select");
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/parameters/parametertypes/filenames/FileNameComponent.java:55]: For file names we use select - so everything fine with me

**Comment by robinschmid** [comment]
Seems fine with me.

**Comment by tomas-pluskal** [comment]
There is a merge conflict

**Pull request merged**